src/components/containers/PanelHeader.js

```diff
@@ -0,0 +1,79 @@
+import Button from 'components/widgets/Button';
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import {PlusIcon, ResizeUpIcon, ResizeDownIcon} from 'plotly-icons';
+import {localize} from 'lib';
+
+class PanelHeader extends Component {
+  getActionButton() {
+    const {action, localize: _, onAction} = this.props;
+    const icon = <PlusIcon />;
+    let className, label;
+
+    if (action === 'addTrace') {
+      label = _('Trace');
+      className = 'js-add-trace-button';
+    }
+
+    if (action === 'addAnnotation') {
+      label = _('Annotation');
+      className = 'js-add-annotation-button';
+    }
+
+    return (
+      <Button
+        variant="primary"
+        className={className}
+        onClick={onAction}
+        icon={icon}
+        label={label}
+      />
+    );
+  }
+
+  render() {
+    const {children, action, allowCollapse, toggleFolds, hasOpen} = this.props;
+    return !children && !action && !allowCollapse ? null : (
+      <div className="panel__header">
+        {children && children.length ? (
+          <div className="panel__header__content">{children}</div>
+        ) : null}
+        <div className="panel__header__actions__container">
+          {allowCollapse ? (
+            <div className="panel__header__collapse" onClick={toggleFolds}>
+              {hasOpen ? (
+                <span>
+                  <ResizeDownIcon />
```

Comment:
The cursor on these collapse/expand buttons is not currently a pointer cursor, but it should be.

Reply:
oh yeah, thanks!